lib/http.js

```diff
@@ -115,7 +114,7 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
             if(!req.user) {
               return callback(null, []);
             }
-            const {actor} = req.user || {};
+            const {actor} = req.user;
```

Comment:
Is there code in another part of the stack that guarantees the existence of `req.user`?

Reply:
Yes, 3 lines above this on L114.  :)